website/blog/2022-12-07-databend-weekly-71.md

```diff
@@ -0,0 +1,147 @@
+
+## What's New
+
+Check out what we've done this week to make Databend even better for you.
+
+### Features & Improvements :sparkles:
+
+**Planner**
+
+- optimize topk in cluser mode ([#9092](https://github.com/datafuselabs/databend/pull/9092))
+
+**Query**
+
+- support `select * exclude [column_name | (col_name, col_name,...)]` ([#9009](https://github.com/datafuselabs/databend/pull/9009))
+- alter table flashback ([#8967](https://github.com/datafuselabs/databend/pull/8967))
+- new table function `read_parquet` to read parquet files as a table ([#9080](https://github.com/datafuselabs/databend/pull/9080))
+- support `select * from @stage` ([#9123](https://github.com/datafuselabs/databend/pull/9123))
+
+**Storage**
+
+- cache policy ([#9062](https://github.com/datafuselabs/databend/pull/9062))
+- support hive nullable partition ([#9064](https://github.com/datafuselabs/databend/pull/9064))
+
+### Code Refactoring :tada:
+
+**Memory Tracker**
+
+- keep tracker state consistent ([#8973](https://github.com/datafuselabs/databend/pull/8973))
+
+**REST API**
+
+- drop ctx after query finished ([#9091](https://github.com/datafuselabs/databend/pull/9091))
+
+### Bug Fixes :wrench:
+
+**Configs**
+
+- add more tests for hive config loading ([#9074](https://github.com/datafuselabs/databend/pull/9074))
+
+**Planner**
+
+- try to fix table name case sensibility ([#9055](https://github.com/datafuselabs/databend/pull/9055))
+
+**Functions**
+
+- vector_const like bug fix ([#9082](https://github.com/datafuselabs/databend/pull/9082))
+
+**Storage**
+
+- update last_snapshot_hint file when purge ([#9060](https://github.com/datafuselabs/databend/pull/9060))
+
+**Cluster**
+
+- try fix broken pipe or connect reset ([#9104](https://github.com/datafuselabs/databend/pull/9104))
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+#### RESTORE TABLE
+
+By the snapshot ID or timestamp you specify in the command, Databend restores the table to a prior state where the snapshot was created. To retrieve snapshot IDs and timestamps of a table, use [FUSE_SNAPSHOT](https://databend.rs/doc/sql-functions/system-functions/fuse_snapshot).
+
+
+```sql
+-- Restore with a snapshot ID
+ALTER TABLE <table> FLASHBACK TO (SNAPSHOT => '<snapshot-id>');
+-- Restore with a snapshot timestamp
+ALTER TABLE <table> FLASHBACK TO (TIMESTAMP => '<timestamp>'::TIMESTAMP);
+```
+
+**Learn More**
+
+- [Docs | RESTORE TABLEt](https://databend.rs/doc/sql-commands/ddl/table/ddl-restore-table)
+- [PR | alter table flashback](https://github.com/datafuselabs/databend/pull/8967)
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+#### Add Build Information to the Error Report
+
+Currently, Databend's error reports consist of error codes and error messages. We would like to add build information to this to help better identify problems.
```

Comment:
Databend currently includes error codes and messages in the error report. When build information is available, troubleshooting will become easier.

Reply:
Thank you for the sample above.

Then can we say sth like this?

An error report currently only contains an error code and some information about why the error occurred. When build information is available, troubleshooting will become easier.